docs/UsingNavigators.md

```diff
@@ -102,11 +102,14 @@ navigator.pop();
 
 A more complete example that demonstrates the pushing and popping of routes could therefore look something like this:
```

Comment:
I feel the same way. We could change `main file` to `index.ios.js or index.android.js` which might be confusing, but probably better?